src/proxy/ClientTable.php

```diff
@@ -107,65 +107,108 @@ public function getOffsetTotal()
-            
-            $progress = 1;
-            for ($i=0; $i<$this->getOffsetTotal(); $i++) {
-                $requestData = $this->request($i);
-                
-                if (!$requestData) {
-                    continue;
-                }
-                
-                $data = array_merge($requestData, $data);
-                gc_collect_cycles();
-                Console::updateProgress($progress++, $this->getOffsetTotal());
-            }
-            
-            Console::endProgress();
-            $this->_contentRows = $data;
-        }
-        
-        return $this->_contentRows;
+	    throw new \ErrorException('Not implemented');
     }
-    
-    public function syncData()
-    {
-        Yii::$app->db->createCommand()->truncateTable($this->getName())->execute();
-        
-        $rows = $this->getContentRows();
 
-        $chunks = array_chunk($rows, 25);
-        
-        foreach ($chunks as $match) {
-            Yii::$app->db->createCommand()->batchInsert($this->getName(), $this->cleanUpBatchInsertFields($this->getFields()), $this->cleanUpMatchRow($match))->execute();
-        }
+    private $_contentRowsCount;
+
+    public function getContentRowCount()
+    {
+    	return $this->_contentRowsCount;
     }
     
-    private function request($offset)
+    public function syncData()
     {
-        $curl = new Curl();
-        $curl->get($this->build->requestUrl, ['machine' => $this->build->machineIdentifier, 'buildToken' => $this->build->buildToken, 'table' => $this->name, 'offset' => $offset]);
-        
-        if (!$curl->error) {
-            $response = Json::decode($curl->response);
-            $curl->close();
-            unset($curl);
-            return $response;
-        } else {
-            $this->build->command->outputError("Error while collecting data from server: " . $curl->error_message);
-        }
-        
-        return false;
+	    try {
+		    if (Yii::$app->db->schema instanceof \yii\db\mysql\Schema) {
+			    // Disable foreign
+			    Yii::$app->db->createCommand('SET SESSION FOREIGN_KEY_CHECKS = 0;')->execute();
+			    Yii::$app->db->createCommand('SET SESSION UNIQUE_CHECKS = 0;')->execute();
+
+			    $sqlMode = Yii::$app->db->createCommand('SELECT @@SQL_MODE;')->queryScalar();
+			    Yii::$app->db->createCommand('SET SESSION SQL_MODE="NO_AUTO_VALUE_ON_ZERO";')->execute();
+		    }
+
+		    Yii::$app->db->createCommand()->truncateTable($this->getName())->execute();
+
+
+		    $this->syncDataInternal();
+
+	    }
+	    finally {
+
+		    if (Yii::$app->db->schema instanceof \yii\db\mysql\Schema) {
+			    Yii::$app->db->createCommand('SET SESSION FOREIGN_KEY_CHECKS = 1;')->execute();
+			    Yii::$app->db->createCommand('SET SESSION UNIQUE_CHECKS = 1;')->execute();
+
+			    Yii::$app->db->createCommand('SET SESSION SQL_MODE=:sqlMode;', [':sqlMode' => $sqlMode])->execute();
+		    }
+	    }
+
     }
+
+	private function syncDataInternal(): void
```

Comment:
currently we are not using php 7 output hinting, please remove. As LUYA still works on php 5.6 but its not active supported anymore.